app/views/vm_common/_live_migrate.html.haml

```diff
@@ -52,6 +52,23 @@
                   'paging_div_buttons_state_enabled' => true,
                   'paging_div_buttons_id'            => "angular_paging_div_buttons",
                   'paging_div_buttons_type'          => "Submit"}
+- unless @explorer
+  %table{:width => '100%'}
+    %tr
+      %td{:align => 'right'}
+        #buttons_on
```

Comment:
Live Migrate and Evacuate are always enabled (hence the code that refers to the button paging div), but I'll add that for the others that need it.